html/pricing/main.handlebars

```diff
@@ -54,6 +54,14 @@
         </div>
       </li>
       <li class="section">
+        <div class="title">> Hyper Func (BETA)</div>
+        <div class="content">
+          <ul>
+            <li>During the beta, Hyper Func is charged at <code>the standard per-second container rates * 100%</code>.</li>
```

Comment:
During the beta, Hyper Func is charged at <code>the standard per-second conatiner rates</code>